requirements.txt

```diff
@@ -1,2 +1,4 @@
 tree-sitter==0.20.1
```

Comment:
Should this file in the root directory?

Reply:
Thought about it and I think so yes. With the Python bindings we might want to define some other dependencies in the future. If this one would be in the `auto-sync` dir, we might end up with a second (duplicate) `requirement.txt`. Which is kinda meh.

Reply:
I see, but how can `./requirements.txt` works if we run `cd bindings/python; python setup.py install`? Seems it will only install all packages in `bindings/python/requirements.txt`.

Reply:
So my thinking was that we should ask the people to setup a Python venv for Capstone. This can be used for the Python binding work and the auto-sync stuff.

Basically:

```bash
python3 -m venv .venv
source ./venv/bin/activate
pip3 install -r requirements.txt # Later, requirements for the bindings come here as well.
cd bindings/python
python setup.py install
```

Just tried it and it works pretty nice.

Reply:
Basicly I think it's ok, but looks kind of mess for user I guess. Installing some pkgs in a command, then installing another pkgs in another command, which can let people confuse.

Reply:
True. What would you prefer?

Reply:
I still believe that the requirements.txt required for user installations and developer installations should be separated for now, unless we have a way to make python's requirements.txt nestable like rust cargo.toml.

Reply:
Is https://github.com/capstone-engine/capstone/pull/2217/commits/debb9d7d59ec5e8e52f4df37cbf7c2c93df55824 fine?

Reply:
Looks good, I'll merged this after ci is done. Thanks.